content/en/docs/concepts/architecture/nodes.md

```diff
@@ -173,18 +173,21 @@ to be unreachable. (The default timeouts are 40s to start reporting
 ConditionUnknown and 5m after that to start evicting pods.) The node controller
 checks the state of each node every `--node-monitor-period` seconds.
 
-In versions of Kubernetes prior to 1.13, NodeStatus is the heartbeat from the
-node. Node lease feature is enabled by default since 1.14 as a beta feature
-(feature gate `NodeLease`, [KEP-0009](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/0009-node-heartbeat.md)).
-When node lease feature is enabled, each node has an associated `Lease` object in
-`kube-node-lease` namespace that is renewed by the node periodically, and both
-NodeStatus and node lease are treated as heartbeats from the node. Node leases
-are renewed frequently while NodeStatus is reported from node to master only
-when there is some change or enough time has passed (default is 1 minute, which
-is longer than the default timeout of 40 seconds for unreachable nodes). Since
-node lease is much more lightweight than NodeStatus, this feature makes node
-heartbeat significantly cheaper from both scalability and performance
-perspectives.
+#### Heartbeats
+
```

Comment:
Hi. I looked at this again. Added a few sentences at the beginning and moved a few more around.
Also created a small list out of the kubelet update info. Not sure about the Lease update interval?
Feel free to use what you like.

Heartbeats, between a node and the control plane, help determine the state of a node.
A heartbeat represents the Node resource `NodeStatus` and the (Lease object) [docs/reference/generated/kubernetes-api/v1.16/#lease-v1-coordination-k8s-io]. Each Node has an associated `Lease` object in the `kube-node-lease` {{< glossary_tooltip term_id="namespace" text="namespace">}}. Compared to the `Node` resource, the `Lease` object is lightweight.

The `kubelet` is responsible for creating and updating the `NodeStatus` and `Lease`.

- The `kubelet` updates the `NodeStatus` either when there is change in status, or
if there has been no update for a configured interval. The default interval
for `NodeStatus` updates is 5 minutes (much longer than the 40 second default timeout for unreachable nodes).
- The `kubelet` creates and updates the `Lease` object. The frequency of the `Lease` updates is independent from the frequency of the `NodeStatus` updates.

With a small footprint heartbeat, the load of maintaining the state of a node is decreased. The `Lease` resource improves the performance of the node heartbeats as the cluster scales.

Reply:
Thanks - I really like it.
I just modified it slightly - please take a look if that still makes sense to you and suggest improvements.